src/components/views/rooms/PinnedEventsPanel.js

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
We do do it elsewhere at the moment, but generally putting components in riot-web that are referenced from react-sdk is a big no-no since it worsens react-sdk's dependency on riot-web.